src/http_server.cc

```diff
@@ -44,8 +44,13 @@
 #define TRITONJSON_STATUSRETURN(M) \
   return TRITONSERVER_ErrorNew(TRITONSERVER_ERROR_INTERNAL, (M).c_str())
 #define TRITONJSON_STATUSSUCCESS nullptr
+#include <unistd.h>  // For sleep
+
 #include "triton/common/triton_json.h"
 
+// #define LOG_VERBOSE_IS_ON ... something like that
+// TODO: hard code log verbose mode to see output in Handle()
```

Comment:
I added a new function to the bindings, which can by doing: `tritonfrontend.enable_logging(2)`
This gives the following output during inference:
![image](https://github.com/user-attachments/assets/ae5b52b5-92a7-43c8-b22a-1e7c064623a7)
This can be made cleaner by adding root level variables in the `tritonfrontend` package such as `MORE_VERBOSE=2, LESS_VERBOSE=1, NO_VERBOSE=0`. Or should we leave the input as integer?

Reply:
Scoped for future PR